packages/@lwc/perf-benchmarks/scripts/build.js

```diff
@@ -126,6 +127,16 @@ function createTachometerJson(htmlFilename, benchmarkName, directoryHash) {
     };
 }
 
+// Given a *.benchmark.js file, extract the exported `cpuThrottlingRate`.
+async function extractCpuThrottlingRate(benchmarkFile) {
+    const contents = await readFile(benchmarkFile, 'utf-8');
+    // Rollup transforms this into `const cpuThrottlingRate = 2; [...] export { cpuThrottlingRate };`,
+    // so we just look for the const, not the export.
+    // We could do a more sophisticated thing than a regex, but the regex works fine.
+    const match = contents.match(/const cpuThrottlingRate = (\d+)/);
```

Comment:
```suggestion
    const match = contents.match(/const cpuThrottlingRate\s*=\s*(\d+)/);
```
In case someone doesn't uses spaces between the variable assignments:

```js
const cpuThrottlingRate=4
```

Reply:
The file will be formatted before committing, but sure, it's possible that Rollup could rewrite it.